osc-collab.py

```diff
@@ -2700,6 +2717,15 @@ def _collab_update(apiurl, username, email, projects, package, ignore_reserved =
         else:
             print 'No configure.{ac,in} information found (tarball is probably not using autotools).'
 
+        if meson_created:
+            meson_basename = os.path.basename(meson)
+            if meson_is_diff:
+                print 'Diff in meson.build between %s and %s is available in %s' % (old_tarball, upstream_tarball_basename, meson_basename)
+            else:
+                print 'Complete meson.build of %s is available in %s' % (upstream_tarball_basename, meson_basename)
+        else:
+            print 'No meson.build information found (tarball is probably not using meson).'
+
```

Comment:
Any reason to not use `mesonopt_created`?

Reply:
oh - indeed; mesonopt_created is missing in the output of what collab did (it's only information to the user not being printed)

Reply:
corrected